shared/TestStandMessages.h

```diff
@@ -61,4 +57,12 @@ typedef struct {
     int32_t y_counts;
 } __attribute__((__packed__)) PositionMsgData;
 
+typedef struct {
+    int32_t temp_ambient;
+    int32_t temp_motor_x;
+    int32_t temp_motor_y;
+    int32_t temp_mpmt;
+    int32_t temp_optical;
```

Comment:
can these be `uint32_t`

Reply:
Could the temperature not be negative?

Reply:
You're right, the thermistor range is actually between -55 to 150. I forgot that it could be negative since I was only imagining above 0 temperatures for the mPMTs.